Yona/Yona/AppDelegate.swift

```diff
@@ -24,7 +26,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         IQKeyboardManager.sharedManager().enable = true
         IQKeyboardManager.sharedManager().enableAutoToolbar = false
         UIApplication.sharedApplication().setStatusBarStyle(UIStatusBarStyle.LightContent, animated: true)
-        
+        NSUserDefaults.standardUserDefaults().setInteger(Tab()!.getTag(Tab.challenges.rawValue), forKey: selectedTab)
+        storyboardName = UIStoryboard(name: "Dashboard", bundle: NSBundle.mainBundle())
+        viewControllers = storyboardName?.instantiateInitialViewController()
```

Comment:
Then we need to discuss these decisions to see the proper ways to approach the needed logic